modules/tensor_mechanics/include/materials/ComputeStrainIncrementBasedStress.h

```diff
@@ -36,10 +36,9 @@ class ComputeStrainIncrementBasedStress : public ComputeStressBase
   const MaterialProperty<RankFourTensor> & _elasticity_tensor;
   /// Old state of the stress tensor material property
   const MaterialProperty<RankTwoTensor> & _stress_old;
-  ///@{ Current and old states of the mechanical strain material property
-  const MaterialProperty<RankTwoTensor> & _mechanical_strain;
+  /// Old of the mechanical strain material property
```

Comment:
Old state of the...

Good catch, by the way!

Reply:
I am surprised that the framework didn't complain about this before, perhaps it's an issue.